meta-iotqa/conf/test/refkit-image-common-extensive.manifest

```diff
@@ -1 +1,3 @@
 # Extensive tests for common profile
+oeqa.runtime.bluetooth.comm_bt_command
```

Comment:
I also re-executed the tests using the latest build from master and noticed bluetoothctl (which the comm_bt_command tests use) is missing from the image, seemingly due to https://github.com/intel/intel-iot-refkit/commit/0dc352e992414fd5cff0a49964649b5e4b1d77c0 and now those tests are failing. According to this, I will need to change them to not depend on it.